requirements-dev.txt

```diff
@@ -19,7 +19,7 @@ pillow
 pre-commit
 pluggy >= 1.4.0
 pytest > 7, < 9 
-pytest-asyncio >= 0.18.2, != 0.22.0, < 0.23.0 # Cannot override event loop in 0.23.0. See https://github.com/pytest-dev/pytest-asyncio/issues/706 for more details.
+pytest-asyncio >= 0.18.2, != 0.22.0, < 0.27.0 # Cannot override event loop in 0.23.0. See https://github.com/pytest-dev/pytest-asyncio/issues/706 for more details.
```

Comment:
The regression has been fixed in 0.26, maybe you want to update your constraints and comments:
https://github.com/pytest-dev/pytest-asyncio/issues/706